src/components/about/About.js

```diff
@@ -114,31 +113,30 @@ const About = () => {
               </span>
             </h2>
             <p>
-              The goal of the Twitterbot on the Blue Witness project is to
-              adequately scrape twitter for reports of incidents of police
+              The goal of the Twitter bot on the Blue Witness project is to
```

Comment:
We need to establish a differentiation here between our Twitter scraper and our Twitter bot. The scraper is responsible for gathering tweets with content specific criteria including (but not limited to) key words such "police" and "misconduct" and their synonyms. It does this multiple times a day (I think every 4 hours) and checks the incoming data against data in our database to prevent duplicates being entered into the database and then passes them to the BERT model to be classified and await administrator approval.

The Twitter bot is a tool that we are currently building that is able to reach out to Twitter users to confirm or supply missing information. It is also triggered by the Administration team and new data gathered awaits approval before being marked as an 'approved' incident report.

Reply:
(I think that the previous Labs cohorts had not fully made this distinction clear in the about section because they had not yet reached that stage of development.)

Reply:
I'm using this to update the section 'Our Twitter Scrapper/Bot'